pages/common/od.md

```diff
@@ -0,0 +1,25 @@
+# od
+
+> Display a file contents in hexadecimal, octal, or printable character format.
+> Optionally display the byte offsets into the file.
+
+- Display file in octal format with the byte offsets in octal as well:
+
+`od --address-radix=o --format=o -v  {{/path/to/file}}`
```

Comment:
Replaced 1st example with the default run, in daily usage it is least used IMO though :) but is the simplest and good to start with I guess.